go/cmd/vttestserver/main.go

```diff
@@ -167,7 +134,44 @@ func parseFlags() (config vttest.Config, env vttest.Environment, err error) {
 	flag.BoolVar(&config.InitWorkflowManager, "workflow_manager_init", false, "Enable workflow manager")
 
 	flag.StringVar(&config.VSchemaDDLAuthorizedUsers, "vschema_ddl_authorized_users", "", "Comma separated list of users authorized to execute vschema ddl operations via vtgate")
+}
 
+func (t *topoFlags) buildTopology() (*vttestpb.VTTestTopology, error) {
```

Comment:
Is this unrelated?

Reply:
No, it was already there, just relocated in the source code as I'm defining the flags in an ```init() ``` function instead of in the ```parseFlags()``` function